main.go

```diff
@@ -27,13 +27,15 @@ import (
 )
 
 var (
-	sync     = time.Second * 30
-	setupLog = ctrl.Log.WithName("setup")
+	sync                    = time.Second * 30
+	setupLog                = ctrl.Log.WithName("setup")
+	namespaceToWatch string = "security-operators"
```

Comment:
Do we want this to be configurable?

Reply:
Is it worth also making reconciliation intervals configurable?

Reply:
Fixed it - although I am not sure about the ENV name I used: `NAMESPACE_TO_WATCH`.